ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -65,14 +66,43 @@ export const ApplicationSummary = (props: {app: models.Application; updateApp: (
             edit: (formApi: FormApi) => (
                 <DataLoader
                     load={() =>
-                        services.clusters.list().then(clusters =>
-                            clusters.map(cluster => ({
-                                title: clusterTitle(cluster),
-                                value: isDestName ? cluster.name : cluster.server
-                            }))
-                        )
+                        Promise.all([
+                            services.clusters.list().then(clusters => clusters.map(cluster => cluster.server).sort()),
+                            services.clusters.list().then(clusters => clusters.map(cluster => cluster.name).sort())
+                        ]).then(([clusterURLs, clusterNames]) => ({clusterURLs, clusterNames}))
                     }>
-                    {clusters => <FormField formApi={formApi} field={fieldDest} componentProps={{options: clusters}} component={FormSelect} />}
+                    {({clusterURLs, clusterNames}) => {
+                        return (
+                            <div className='row'>
+                                {(destFormat.toUpperCase() === 'URL' && (
```

Comment:
Text box is not aligned correctly ( probably because of dropdown )

![image](https://user-images.githubusercontent.com/426437/88944076-24c7c300-d241-11ea-9b43-bb7c5fd54f21.png)

Reply:
Fixed: <img width="987" alt="Screen Shot 2020-07-31 at 2 45 17 AM" src="https://user-images.githubusercontent.com/62673041/89033766-65304b00-d2ec-11ea-84a4-906a5d800ed1.png">